src/components/ObjectInfo.vue

```diff
@@ -29,6 +29,10 @@
       <i class="fas fa-fw fa-horse" style="text-shadow: lightblue 0px 0px 5px; color: lightblue;"></i>
       Lord of the Mountain
     </section>
+    <section class="search-result-rankup" v-if="data.field_area == 28">
+      <i class="fas fa-fw fa-ban" style="color:tomato"></i>
+      No rankup on Eventide Island
```

Comment:
should be "No scaling"; "rankup" refers to the Master Mode levelling up mechanic

Reply:
So, would `No scaling on Eventide` be ok?

Reply:
Yeah, that sounds fine.